ui-components/src/page_components/NavigationHeader.tsx

```diff
@@ -42,11 +43,19 @@ const NavigationHeader = ({
       {
         label: tabs?.listingLabel || "",
         path: `/listings/${listingId}`,
+        activePaths: [`/listings/${listingId}`],
         content: undefined,
       },
       {
         label: tabs?.applicationsLabel || "",
         path: `/listings/${listingId}/applications`,
+        activePaths: [
```

Comment:
Definitely a problem for different PR, but eventually we'll need to strip these implementation details out of ui-components

Reply:
https://github.com/bloom-housing/bloom/issues/2962